next.config.js

```diff
@@ -1,4 +1,9 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
-  reactStrictMode: true
+  reactStrictMode: true,
+  images: {
+    domains: [
+      "res.cloudinary.com"
+    ]
+  }
```

Comment:
```suggestion
```

This shouldn't be needed once you use next/image instead of the Chakra image component.

Reply:
Even with next/image this still has to be defined.
But I'll try to find a way around it.

Reply:
> Even with next/image this still has to be defined. But I'll try to find a way around it.

Just tried it and it worked without a problem with this lines gone. Shouldn't be a problem!

Reply:
sir here is the error 
that was why i prevented it with does lines i'd advice we keep it there so as not to cause any future errors
![image](https://user-images.githubusercontent.com/72051159/218285773-abcbff79-5138-4e54-a4d2-09dd5c97e1ce.png)

Reply:
But if you insist 
i guess i'll just remove the cloudinary link so it'll loads from the codebase 

what do you think sir?

Reply:
I think it's the best way so far. I'm just very wary of hard-coding a domain into the project if we can avoid it honestly.

Could you elaborate on what you did to get that error?

Reply:
` <Image src={"https://res.cloudinary.com/phantom1245/image/upload/v1675875866/techishiring/techish_qcfih9.png" || contactImg}  alt="Contact Form Image" width={1000}  height={1000} />`

Reply:
Hey @dev-phantom. So I checked out your branch with your updated changes and when I made the changes I suggested, it seems to run fine. Could you make the suggested changes please?

Reply:
No problem